to_do_list_part_two.py

```diff
@@ -27,11 +27,14 @@ def view_tasks(task_list):
         print("The to-do list is empty. Consider adding a new task!")
     else:
         print("To-Do List:")
-        priority_order = {'high': 1, 'medium': 2, 'low': 3}
-        sorted_tasks = sorted(task_list, key=lambda x: (priority_order[x['priority']], x['deadline']))
+        sorted_tasks = sort_tasks_by_priority_and_deadline(task_list)
         for idx, task in enumerate(sorted_tasks, start=1):
             print(f"{idx}. {task['task']} - {task['priority']} - {task['deadline'].strftime('%Y-%m-%d')}")
```

Comment:
```suggestion
            print(f"{idx}. {task['name']} - {task['priority']} - {task['deadline'].strftime('%Y-%m-%d')}")
```

We decided the contents of a dictionary like this
```
{
    "name": "task_a",
    "priority": "high",
    "deadline": "2024-10-15"
}
```

Reply:
`task['deadline']` is string type. Not datetime type.
So it doesn't have method `strftime`.

Did you convert it to datetime type anywhere?